packages/components/src/simple-select-control/style.scss

```diff
@@ -0,0 +1,122 @@
+.woocommerce-simple-select-control__dropdown {
+	position: relative;
+	width: 100%;
+	height: 56px;
+	border: 1px solid #b0b5b8;
+	box-shadow: none;
+	box-sizing: border-box;
+	border-radius: 3px;
+	background: #fff;
+	font-size: 16px;
+	line-height: 54px;
+	font-weight: 400;
+	margin-top: $gap;
+	margin-bottom: $gap;
+
+	.woocommerce-simple-select-control__selector {
+		&.components-button {
+			border: 0;
+			background: transparent;
+			box-shadow: none;
+			justify-content: unset;
+			margin-left: 0;
+			position: absolute;
+			top: 0;
+			left: 8px;
+			width: calc(100% - 24px);
+
+			&:focus:not(:disabled) {
+				box-shadow: none;
+				outline: none;
+				background-color: transparent;
+				outline-offset: initial;
+			}
+
+			&::after {
+				display: block;
+				pointer-events: none;
+				position: absolute;
+				float: right;
+				line-height: 56px;
+				font-family: dashicons, sans-serif;
+				font-size: 20px;
+				content: '\f140';
+				z-index: 101;
+				height: 24px;
+				width: 24px;
+				margin-top: 0;
+				top: 0;
+				right: 0;
+				bottom: 16px;
+				color: #000;
+			}
+		}
+	}
+
+	.woocommerce-simple-select-control__dropdown-content {
+		&.components-popover.is-bottom {
+			margin-top: -$gap-largest * 2;
+			z-index: 999;
+
+			&::before,
+			&::after {
+				display: none;
+			}
+		}
+
+		.components-button {
+			display: block;
+			position: relative;
+			padding: 10px 20px 10px 40px;
+			width: 100%;
+			text-align: left;
+
+			&.is-selected {
+				background: $muriel-gray-100;
+			}
+		}
+	}
+
+	&.is-empty {
+		.woocommerce-simple-select-control__selector {
+			&.components-button {
+				background: #fff;
+				font-size: 16px;
+				line-height: 54px;
```

Comment:
Nice!  Looks good, but tiny issue when an option is selected:

<img width="504" alt="Screen Shot 2019-07-10 at 10 56 27 AM" src="https://user-images.githubusercontent.com/10561050/60937365-b85ee680-a302-11e9-9541-a6167ab8fbbc.png">

Not sure if the line-height fix will work for this, but we can also use flexbox as an alternative:

```scss
    height: 100%;
    display: flex;
    justify-content: center;
    flex-direction: column;
```